internal/ingress/controller/config/config.go

```diff
@@ -754,6 +762,9 @@ func NewDefault() Configuration {
 	defNginxStatusIpv6Whitelist := make([]string, 0)
 	defResponseHeaders := make([]string, 0)
 
+	defAnnotationValueWordBlocklist := []string{"load_module", "lua_package", "_by_lua", "location", "root"}
```

Comment:
should we add others like token etc?

Reply:
you are fast uh! haha! Yeah, this list should be updated I guess. I tried to add the dangerous ones!

Reply:
So what list do you think it's good to have here? When you say token, you say like the whole "/var/run/..." for the serviceaccount?

Reply:
I would also add probably proxy_pass directives, so someone cannot bypass the internal balancer for the evil :D